src/cascade/input_data/db/mortality.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+
+from cascade.core.db import connection
+
+
+def get_excess_mortality_data(execution_context):
+    model_version_id = execution_context.parameters.model_version_id
+
+    query = """
+     SELECT
+                model_version_dismod_id as data_id,
+                location_id,
+                year_start as time_lower,
+                age_start as age_lower,
+                age_end as age_upper,
+                sex_id as x_sex,
+                mean as meas_value,
+                lower as meas_lower,
+                upper as meas_upper,
+                standard_error
+            FROM
+                epi.t3_model_version_emr t3_emr
+            WHERE model_version_id = %(model_version_id)s
+            AND t3_emr.outlier_type_id = 0
+    """
+
+    with connection(execution_context) as c:
+        data = pd.read_sql(query, c, params={"model_version_id": model_version_id})
```

Comment:
I am wondering which of these c is: (from pandas.read_sql doc), a _ConnectionFairy?
    con : SQLAlchemy connectable (engine/connection) or database string URI or DBAPI2 connection (fallback mode)
        Using SQLAlchemy makes it possible to use any DB supported by that library. 
        If a DBAPI2 object, only sqlite3 is supported.

Reply:
You can answer this question either by tracing the code back to where the connection get's created (there's a description of what it is in the ezfuncs code)  or by using a debugger to examine the live object. If you aren't familiar with debuggers I can show you how to work with the python one. It's a useful tool to have access to.

Reply:
I may have another question, that I am not asking here.   I did look at the type previously, and saw ConnectionFairy.